### ISSUE TITLE
gem.cbc.ca

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#3896]
### Basic Information

**Test link:** `https://gem.cbc.ca/media/schitts-creek/season-6/episode-8/38e815a-01245275b28`
**Category:** breakage
**Reported from:** Canada
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.145

### Additional Message

Blocks auto-play next episode
Blocks play next episode button and play next episode on pop-up at end of videos

**Comment by mapx-** [comment]
working fine on my end using default settings in uBO

**Comment by jspenguin2017** [comment]
Yep, next episode functionalities are working properly for me as well.
